task.go

```diff
@@ -87,29 +87,21 @@ type Task struct {
 // Start will mark the task as started by populating
 // Started.
 //
-// If Started is already populated then the existing
-// value is returned.
 func (t *Task) Start() time.Time {
-	if t.started.IsZero() {
-		t.started = time.Now().In(time.UTC)
-		t.Started = t.started.Format(time.RFC3339)
-	}
+	t.started = time.Now().In(time.UTC)
+	t.Started = t.started.Format(time.RFC3339)
 
 	return t.started
 }
 
 // End will mark the task as started by populating
 // Ended and setting Result.
 //
-// If Ended is already populated then the existing
```

Comment:
not modifying the result msg and end stamp after a task has already run is unexpected behavior and hard to track down. If the task has run it should always return the result, message and end time even if those values are already present.